### PR TITLE
chore: Update pull request template

### PR DESCRIPTION
**Pull request by suddjian**
### SUMMARY

Just a minor change to make the PR template a little clearer

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
 ### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
 <!--- Skip this if not applicable -->
 
-### TEST PLAN
-<!--- What steps should be taken to verify the changes -->
+### TESTING INSTRUCTIONS
```
Comment (junlincc) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: TESTING INSTRUCTIONS(required!)

Comment (suddjian) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: Added this to the comment below the header

**Review by pkdotson — APPROVED** [review]
LGTM!

**Review by junlincc — COMMENTED** [review]
product sign-off ✅

**Pull request merged**